src/app/modules/main/chat_section/controller.nim

```diff
@@ -93,6 +93,15 @@ method init*(self: Controller) =
     var args = ContactArgs(e)
     self.delegate.onContactUnblocked(args.contactId)
 
+  self.events.on(SIGNAL_CHAT_UPDATE) do(e: Args):
+    var args = ChatUpdateArgsNew(e)
+
+    for chatDto in args.chats:
```

Comment:
@borismelnik if you don't mind, please move this logic to `module.nim` just to be uniform everywhere.
```
self.events.on(SIGNAL_CHAT_UPDATE) do(e: Args):
    var args = ChatUpdateArgsNew(e)
    self.delegate.addChatsIfDontExist(chats: seq[ChatDto])
```

Reply:
no problem :)